barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/arithmetization.hpp

```diff
@@ -104,6 +102,8 @@ template <typename FF, size_t NUM_WIRES, size_t NUM_SELECTORS> class ExecutionTr
         }
     }
 #endif
+  private:
+    uint32_t fixed_size = 0; // Fixed size for use in structured trace
```

Comment:
made this private to avoid accidentally getting it. Need to call get_fixed_size() instead because it's 0 in the unstructured case.